pkg/scheduler/invoker/localinvoker_test.go

```diff
@@ -65,7 +65,11 @@ func runLocalReconciler(t *testing.T, simulateError bool) (reconciliation.Reposi
 		SchedulingID: reconEntity.SchedulingID,
 	})
 	require.NoError(t, err)
-	require.Len(t, opEntities, 7)
+	if clusterStateMock.Status.Status.IsDeletion() {
```

Comment:
Shoulnd't it be `IsDeleteCandidate` ?

Reply:
It is actually correct to have `isDelettionInProgress()` since when `run` and `runOnce` are called the status should be always either `Reconciling` or `Deleting`.

What is wrong is the test status created in `testClusterState()`. It needs to be changed to an in progress status.

Reply:
I think as of now i check on isDelettionInProgress(). Nevertheless Im not sure where to change the test status.

Reply:
I changed the test behavior as discussed